controllers/routemonitor/adder/adder_test.go

```diff
@@ -108,13 +111,22 @@ var _ = Describe("Adder", func() {
 			// Arrange
 			get = helper.NotFoundErrorHappensOnce()
 			create.CalledTimes = 1
+
 		})
 		When("the resource Exists", func() {
 			BeforeEach(func() {
 				// Arrange
 				get.ErrorResponse = nil
 				create.CalledTimes = 0
+				mockClient.EXPECT().Status().Return(mockStatusWriter).Times(1)
```

Comment:
I don't really understand the relation to the code change

Reply:
If you don't call the status function you get an unexpected call in testing since I added that one function to the EnsureServiceMonitorResourceExists